test/e2e/basic_test.go

```diff
@@ -101,7 +101,7 @@ func (data *TestData) testDeletePod(t *testing.T, podName string, nodeName strin
 		t.Fatalf("Expected 1 pod interface, got %d", len(podInterfaces))
 	}
 	ifName := podInterfaces[0].InterfaceName
-	podIP := podInterfaces[0].IP
+	podIPs := strings.Split(strings.ReplaceAll(podInterfaces[0].IP, " ", ""), ",")
```

Comment:
nit: you could use the following here since the separator in `Split` can be a string:
```go
podIPs := strings.Split(podInterfaces[0].IP, ", ")
```

Reply:
Thank you.